tests/trainer/flags/test_fast_dev_run.py

```diff
@@ -19,3 +25,77 @@ def test_skip_on_fast_dev_run_tuner(tmpdir, tuner_alg):
     expected_message = f'Skipping {tuner_alg} since fast_dev_run is enabled.'
     with pytest.warns(UserWarning, match=expected_message):
         trainer.tune(model)
+
+
+@pytest.mark.parametrize('fast_dev_run', [1, 4])
+@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
+def test_callbacks_and_logger_not_called_with_fastdevrun(tmpdir, fast_dev_run):
+    """
+    Test that ModelCheckpoint, EarlyStopping and Logger are turned off with fast_dev_run
+    """
+    class FastDevRunModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+            self.training_step_called = False
+            self.validation_step_called = False
+            self.test_step_called = False
+
+        def training_step(self, batch, batch_idx):
+            self.log('some_metric', torch.tensor(7.))
+            self.logger.experiment.add_scaler('some_distribution', torch.randn(7) + batch_idx)
```

Comment:
```suggestion
            self.logger.experiment.dummy_log('some_distribution', torch.randn(7) + batch_idx)
```
did you misspell "add_scalar" on purpose? let's make it very obvious that we deal with a dummy logger here :))